tests/resources/sharepoint/nsb2023/raw/list_item10.json

```diff
@@ -49,7 +49,7 @@
   "CraniotomyType": "3mm",
   "ImplantIDCoverslipType": "3mm stacked coverslip",
   "Burr_x0020_hole_x0020_1": null,
-  "Inj1VirusStrain_rt": "Select...",
+  "Inj1VirusStrain_rt": null,
```

Comment:
Why are the raw data resource files being update? Did NSB change the responses they're returning?

Reply:
Virus Strain is a text input, not a dropdown